components/pages/explorer/Jbrowse/JbrowseWrapper.tsx

```diff
@@ -216,31 +242,31 @@ const JbrowseWrapper = () => {
             selectedFiles={jbrowseInput}
           />
           <JbrowseSelectedFilesTable />
-          {jbrowseLoading && (
-            <div
-              css={css`
-                width: 100%;
-                height: 100%;
-                min-height: 500px;
-                display: flex;
-                position: absolute;
-                justify-content: center;
-                padding-top: 200px;
-                top: 0;
-                left: 0;
-                bottom: 0;
-                right: 0;
-                background: rgba(255, 255, 255, 0.8);
-                z-index: 999;
-              `}
-            >
-              <Spinner width={50} />
-            </div>
-          )}
+          {jbrowseLoading && <Loader />}
         </>
       )}
     </div>
   );
 };
 
+const JbrowseWrapper = () => {
+  // handle compatibility check before trying to load jbrowse,
+  // in case the user comes to the jbrowse tab with an invalid selection.
+  const { jbrowseErrorText, jbrowseLoading } = useJbrowseCompatibility();
+
+  return jbrowseLoading ? (
+    <Loader />
+  ) : !!jbrowseErrorText ? (
```

Comment:
implicit value evaluation is good enough in this case.
e.g. if the string is not empty, it'll be truthy, otherwise falsy.
```suggestion
  ) : jbrowseErrorText ? (
```